CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Wazuh Docker v3.11.0_7.5.1
```

Comment:
Please include in the Changelog the merged PRs #261 and #271

Reply:
Added those changes